src/datasource/__tests__/integration.test.ts

```diff
@@ -304,5 +307,29 @@ describe(DBDataSource, () => {
       const result = await ds.idLoader.load(newRow1.id)
       expect(result).toMatchObject(newRow1)
     })
+
+    it('can set query options on a loader', async () => {
+      const row1: DummyRowType = {
+        id: 19,
+        code: 'any',
+        name: 'any',
+        withDefault: 'asdf',
+        tsTest: new Date('2020-12-05T00:00:00.001Z'),
+        jsonbTest: { a: 1 },
+      }
+      const row2: DummyRowType = {
+        id: 20,
+        code: 'any',
+        name: 'any',
+        withDefault: 'asdf',
+        tsTest: new Date('2020-12-05T00:00:00.002Z'),
+        jsonbTest: { a: 1 },
```

Comment:
Minor: sometimes it's nice when test data is easily distinguishable, for example `jsonbTest: { a: 2 }` for row 2, things like that.